lib/main.dart

```diff
@@ -0,0 +1,144 @@
+import 'package:flutter/material.dart';
+import 'new_todo.dart';
```

Comment:
nit: add a blank line between the package import block and the relative import block.